ci/upload_to_owncloud.sh

```diff
@@ -29,6 +29,52 @@ usage() {
     echo ""
 }
 
+create_dir() {
+    # create dir $1 in $OWNCLOUD_URL/$user/.
+    # if $2 is true, try to delete any existing dir first.
+    #
+    # note: parent directories are not created automatically, so make sure
+    #       to create them if they don't exist yet.
+    #
+    # uses:
+    #   OWNCLOUD_URL
+    #   user
+    local dir delete status
+    dir="$1"
+    delete="$2"
+    status=0
+
+    dbg "Create directory: $dir"
+    if [ "$delete" = true ] ; then
+        curl ${QUIET:+-s -S} -f -n -X DELETE "$OWNCLOUD_URL/$user/$dir" > /dev/null 2>&1
```

Comment:
Given the number of times `curl ${QUIET:+-s -S} -f -n -X ` is repeated, it might make sense to make a wrapper for that as well (nitpick).